_sass/sidebar.scss

```diff
@@ -302,14 +302,18 @@
   }
 }
 
```

Comment:
I had not getting noticed those buttons were hiding to the right. Not sure what you were meant to, that would be "TOTAL DISRESPECT" for laptop user!